Catalog/Application/AppDelegate.swift

```diff
@@ -11,12 +11,15 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
+    private let securityProtectionManager = SecurityProtectionManager()
 
     func application(
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
     ) -> Bool {
+        securityProtectionManager.checkSystemStatus()
```

Comment:
I would swap lines `20` and `21` to have `securityProtectionManager` logic one after another, but still, it can stay like that.